WaterLog_Backend/Controllers/TankReadingsController.cs

```diff
@@ -14,37 +14,50 @@ namespace WaterLog_Backend.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
-    public class TankLevelsController : ControllerBase
+    public class TankReadingsController : ControllerBase
     {
         private readonly DatabaseContext _db;
         readonly IConfiguration _config;
-        public TankLevelsController(DatabaseContext context, IConfiguration config)
+        public TankReadingsController(DatabaseContext context, IConfiguration config)
         {
             _db = context;
             _config = config;
         }
 
         // GET api/levels
         [HttpGet]
-        public async Task<ActionResult<IEnumerable<TankLevelsEntry>>> Get()
+        public async Task<ActionResult<IEnumerable<TankReadingsEntry>>> Get()
```

Comment:
Please use more descriptive end points

Reply:
This is just a simple get method? What more do i need to elaborate on?

Reply:
What are you getting? 
`getTankReading` or `getTankblabla`
just `get` is not good enough

Reply:
alright cool, made it more specific